bot/__main__.py

```diff
@@ -1,45 +1,107 @@
+"""The main Hotwired file."""
```

Comment:
There is no need for this comment, this is obvious from the name `__main__.py`
```suggestion
```

Reply:
D100 : Missing module docstring
What do you thing should be included in the docstring ? I didn't have any idea so I stated the obvious

Reply:
Nothing, we don't respect `D100` error, it's not in respected defaultly by `flake8`, in fact we don't have module docstrings anywhere in the code

Reply:
You should respect the MIT license :
`The above copyright notice and this permission notice shall be included in all
copies or substantial portions of the Software.`